mi/reporting/actions.py

```diff
@@ -0,0 +1,111 @@
+import csv
+import re
+from datetime import date, datetime
+from pathlib import Path
+from typing import Generator
+
+from helpers.log import log
+from mi.reporting.paths import PATH_TO_REPORT_CSV
+from mi.reporting.resources import (
+    each_sql_statement,
+    get_credentials,
+    get_endpoint,
+    get_lambda_name,
+    make_report_path,
+)
+from mi.sql_query.model import Response, Sql, SqlQueryEvent, Status
+
+SQL_SELECT_REGEX = re.compile(r"SELECT(.*)FROM", flags=re.DOTALL)
+SQL_SELECT_SEPARATOR = ","
+SQL_ALIAS_SEPARATOR_REGEX = re.compile(r"as|AS")
+
+
+@log("Created query events")
+def each_query_event(
```

Comment:
to make it clear these are running stored queries - maybe have this named more like `each_stored_report_query`?